### PR TITLE
Add README.md; Cleanup gce.ini

### PR DESCRIPTION
**Pull request by ifigotin**
Added README.md to explain how ansible test is working.
Removed unused variables in gce.ini.

### CODE REVIEW

File: e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+End to End test uses Ansible playbooks to accoumplish the following:
```
Comment (ludoch) [inline e2e/README.md]: accomplish?

File: e2e/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+* Grab and build the lastest gcloud-maven-plugin code from Github
+* Grab and install the lastest released gcloud sdk
```
Comment (ludoch) [inline e2e/README.md]: latest (here and above)

Comment (ifigotin) [inline e2e/README.md]: Done.

File: e2e/README.md (hunk 1 above)
Comment (ifigotin) [inline e2e/README.md]: Done.

**Comment by ludoch** [comment]
LGTM

**Pull request merged**